Licensing: Add missing property definitions (#225)
spdx/spdx-3-model: model/Licensing/Properties/seeAlso.md

## Summary

A seeAlso contains a URL where the License or LicenseException can be found
in use.

## Description

A seeAlso defines a cross-reference with a URL where the License or

> LicenseException can be found in use by one or a few projects.

Reviewer: ```suggestion
LicenseException can be found in use by one or more projects.
```

Author: I think "a few" is correct here. The general suggestion is that a few cross-references are appropriate, but doesn't need to list every project that uses this license.
